payjoin-cli/src/app/config.rs

```diff
@@ -68,10 +68,10 @@ impl Config {
             if matches.get_flag(flag) {
                 if selected_version.is_some() {
                     return Err(ConfigError::Message(format!(
-                        "Multiple version flags specified. Please use only one of: {}",
-                        Self::VERSION_FLAGS
+                        "Multiple version flags specified. Please use only one of: {flags}",
+                        flags = Self::VERSION_FLAGS
                             .iter()
-                            .map(|(flag, _)| format!("--{}", flag))
+                            .map(|(flag, _)| format!("--{flag}"))
```

Comment:
I'd rather we just `Self::VERSION_FLAGS` here and droped the additional var

Reply:
Removed the flag variable. Clippy is ok with non-inline vars if you are using a expression